indico/MaKaC/webinterface/pages/contributions.py

```diff
@@ -190,84 +192,73 @@ def _getBody(self, params):
 class WPContributionModifBase(WPConferenceModifBase):
 
     def __init__(self, rh, contribution, **kwargs):
-        WPConferenceModifBase.__init__(self, rh, contribution.getConference(), **kwargs)
+        WPConferenceModifBase.__init__(self, rh, contribution.event_new.as_legacy, **kwargs)
         self._contrib = self._target = contribution
         from MaKaC.webinterface.rh.reviewingModif import RCPaperReviewManager
         self._isPRM = RCPaperReviewManager.hasRights(rh)
-        self._canModify = self._contrib.canModify(rh.getAW()) or (self._contrib.getSession() and self._contrib.getSession().canCoordinate(rh.getAW(), "modifContribs"))
+        self._canModify = (self._contrib.can_manage(session.user) or
```

Comment:
`self._contrib.can_manage(session.user)` should be enough (if the user is a session coordinator who can manage contributions `can_manage` returns true)